### PR TITLE
[do not merge] Add a basic yast2-network test

### PR DESCRIPTION
**Pull request by imobach**
We're working on fixing [bsc#916013](https://bugzilla.suse.com/show_bug.cgi?id=916013) and this test should be included when the bug is closed. So, please, don't merge it yet.

Anyway, feedback is welcome. :)

### CODE REVIEW

File: tests/x11/yast2_network.pm [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  # Start xterm as root
+  x11_start_program("xterm");
+  wait_idle;
+  become_root;
+
+  # Make sure that firewall is running
+  script_run "yast2 firewall enable";
+
+  # Enable routing
+  $self->start_yast2_lan();
```
Comment (coolo) [inline tests/x11/yast2_network.pm]: JFYI (assuming you're not perl native :) if you don't need the object in the function, you can skip the $self. $self->start is just syntactic sugar for start($self)

Comment (imobach) [inline tests/x11/yast2_network.pm]: Yes, I'm not a perl native :) Thanks for the tip, I didn't know it.

File: tests/x11/yast2_network.pm [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  # Stop firewall
+  script_run "yast2 firewall disable";
+
+  # Check that routing is still enabled
+  check_routing_enabled_console();
+  start_yast2_lan();
+  check_routing_enabled_ui();
+
+  # Success
+  $self->clean_and_quit;
```
Comment (lnussel) [inline tests/x11/yast2_network.pm]: not defined?

**Comment by imobach** [comment]
@lnussel You are right. I've made some fixes and I've rebased against master. Anyway, it's not ready to be merged yet.

Thanks!

File: tests/x11/yast2_network.pm [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+sub start_yast2_lan() {
+    script_run "yast2 lan";
+    assert_screen "yast2_network-lan", 50;
+}
+
+# Test for basic yast2-network functionality.
+sub run() {
+  my $self = shift;
```
Comment (lnussel) [inline tests/x11/yast2_network.pm]: $self not needed anymore :)

Comment (imobach) [inline tests/x11/yast2_network.pm]: Ugh, dead code. Fixed. Thanks.

**Comment by coolo** [comment]
when is it ready? it hangs around for a while now

**Comment by imobach** [comment]
@coolo Yes, you're right. I'll make sure during this week that it works ok and then I'll ask you to merge it.

Thanks!

**Comment by coolo** [comment]
done?

**Comment by imobach** [comment]
@coolo Hi, as it looks like the way to test YaST modules on OpenQA is changing, I'd prefer closing and I'll re-submit it when done.

**Pull request closed without merging**